src/Orc.FileAssociation/Services/FileAssociationService.cs

```diff
@@ -102,5 +105,15 @@ public async Task UndoAssociationFilesWithApplicationAsync(ApplicationInfo appli
             }
 
         }
+
+        public async Task OpenPropertiesWindowForExtensionAsync(string extension, IFileService fileService)
+        {
+            var appPath = AppDomain.CurrentDomain.BaseDirectory;
+            var fileName = $"Click on 'Change' to select default {extension} handler.{extension}";
+            var finalPath = Path.Combine(appPath, fileName);
+
+            fileService.Create(finalPath);
```

Comment:
Missed using

Reply:
Add logging so we can debug this later on production machines